config/rekor/rekor/300-rekor.yaml

```diff
@@ -19,7 +19,11 @@ spec:
       serviceAccountName: rekor
       containers:
       - name: rekor
-        image: gcr.io/projectsigstore/rekor-server@sha256:af91aee88c6d8efb3175b441420a4c61691ec5d6a72303c8ef47750c1dfd2325 # v0.11.0
+        # This has been built from here to pick up the file based signer change:
+        # https://github.com/sigstore/rekor/actions/runs/3061668501/jobs/4941722492
+        # Which contains this change that we need.
+        # https://github.com/sigstore/rekor/pull/1049
+        image: gcr.io/projectsigstore/rekor/ci/rekor/rekor-server@sha256:91a6fa93804e1c0b48942a6a0e38b30b851a9f281a65e84a05cd85c138450d7f
```

Comment:
todo to switch back to gcr.io/projectsigstore/rekor-server with the next release?

Reply:
Yup, great call!
https://github.com/sigstore/scaffolding/issues/359